.circleci/test-scripts/baseline/compare-and-update-baseline.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
Might be worth it to add `-u` in order for the script to fail explicitly if some env variable isn't defined.
```suggestion
set -euo pipefail
```